packages/server/package.json

```diff
@@ -33,7 +33,7 @@
     "@types/dotenv": "^6.1.1",
     "@types/fbjs": "^3.0.2",
     "@types/graphql": "^14.5.0",
-    "@types/ioredis": "^4.0.10",
+    "@types/ioredis": "^4.28.10",
```

Comment:
The [migration guide](https://github.com/luin/ioredis/wiki/Upgrading-from-v4-to-v5) says we can now remove `@types/ioredis`. I think we can create an issue about this if make sense.

Reply:
Oh nice! I'd prefer to handle it in a separate PR. I've created an issue here: https://github.com/ParabolInc/parabol/issues/7280